tests/data.tf

```diff
@@ -0,0 +1,133 @@
+data "aws_partition" "current" {}
```

Comment:
This is not used, might as well remove it.

Reply:
removed